Formula/minicom.rb

```diff
@@ -55,6 +67,14 @@ def caveats
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/minicom -v", 1)
```

Comment:
Just leave the test as it was. We don't need it to pass for head installs.

Reply:
ok, reverted this